src/components/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+// categoriesSlice.js
+/* eslint-disable */
+
+import { createSlice } from '@reduxjs/toolkit';
+
```

Comment:
- [ ] Kindly consider defining CHECK_STATUS as action types, using the [ducks pattern](https://github.com/erikras/ducks-modular-redux),
e.g `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`

Reply:
Implemented 👍